packages/w3up-client/src/client.js

```diff
@@ -250,15 +252,23 @@ export class Client extends Base {
    * If an account is not provided, the space is created without any delegation and is not saved, hence it is a temporary space.
    * When an account is provided in the options argument, then it creates a delegated recovery account
    * by provisioning the space, saving it and then delegating access to the recovery account.
+   * In addition, it authorizes the listed Content Serve Services to serve content from the created space.
+   * It is done by delegating the `space/content/serve/*` capability to the Content Serve Service.
+   * User can skip the Content Serve authorization by setting the `skipContentServeAuthorization` option to `true`.
    *
-   * @typedef {object} CreateOptions
-   * @property {Account.Account} [account]
+   * @typedef {object} SpaceCreateOptions
+   * @property {boolean} [skipContentServeAuthorization] - Whether to skip the Content Serve authorization. It means that the content of the space will not be served by any Content Serve Service.
+   * @property {`did:${string}:${string}`[]} [authorizeContentServeServices] - The DID Key or DID Web of the Content Serve Service to authorize to serve content from the created space.
+   * @property {import('./types.js').ConnectionView<import('./types.js').ContentServeService>} [connection] - The connection to the Content Serve Service that will handle, validate, and store the access/delegate UCAN invocation.
```

Comment:
Each content serve service will have a different URL. For the `createSpace` function I think we should just require a DID and a URL rather than a connection i.e. something like `authorizeContentServeServices: Array<{ serviceDID: DID, serviceURL: URL }>` option. It's just easier to use. We can then create the connection object internally for `authorizeContentServe`.

I wonder if it is clearer to have `skipGatewayAuthorization` and `authorizeGatewayServices` and we add bitswap as seperate options (`skipPeerAuthroization`, `authorizePeerServices`) later? WDYT?

Reply:
@alanshaw what about `authorizeContentServeServices: Array<{ serviceDID: DID, connection: ConnectionView }>`, the reason for that is: it is better for testing because then we can easily mock the connection. WDYT?

> I wonder if it is clearer to have skipGatewayAuthorization and authorizeGatewayServices and we add bitswap as seperate options (skipPeerAuthroization, authorizePeerServices) later? WDYT?

I think it will be better. Easier to understand.

Reply:
Ah yes makes sense, what about `Array<ConnectionView | { serviceDID: DID, serviceURL: URL }>`?

Reply:
Instead of using the `skipGatewayAuthorization` I think it is simple if we just check the array.

Reply:
Ok. I've moved forward with the `Array<ConnectionView>` option.